zebrad/src/components/sync/end_of_support.rs

```diff
@@ -22,8 +22,8 @@ pub const ESTIMATED_RELEASE_HEIGHT: u32 = 2_562_000;
 ///
 /// - Zebra will exit with a panic if the current tip height is bigger than the `ESTIMATED_RELEASE_HEIGHT`
 ///   plus this number of days.
-/// - Currently set to 16 weeks.
-pub const EOS_PANIC_AFTER: u32 = 112;
+/// - Currently set to 14 weeks.
+pub const EOS_PANIC_AFTER: u32 = 70;
```

Comment:
It should panic at height 2706640, the expected NU6 activation height on Mainnet is 2726400, do we want to lower the end of service height a little further, or is this okay?

Reply:
That's ~2 weeks before, right? It seems enough time

Reply:
> That's ~2 weeks before, right?

It should be closer to 4 weeks